### ISSUE TITLE
SpringBoot: some controllers that don't use Katharsis?

### ISSUE THREAD

**Issue by adhamhf** [katharsis-project/katharsis-framework#419]
Is it possible have controllers in a SB app that use Katharsis and some that don't?   I need one controller endpoint that completely bypasses katharsis because it is x-www-form-urlencoded based.  As soon as I `@Import({KatharsisConfigV3.class})` in my application class or in separate Katharsis controller the x-www-form-urlencoded endpoint stops working.  It looks like the Katharsis filter is kicking in reading the request body, which means the body is empty when Spring tries to decode it as a form based request.

Thoughts?

**Comment by exports** [comment]
May be you could modify the filter order by manual register the katharsis filter with `FilterRegistrationBean`?

**Comment by jianglibo** [comment]
When using KatharsisConfigV3, Katharsis works as a standard servlet filter. So katharsis dominate the URL space you defined with "pathPrefix". I think using a space other than "/" will help. For instance "/api", limit the Katharsis functions only on this space only. out that space, do anything you use to do.

**Comment by adhamhf** [comment]
@jianglibo I have `katharsis.pathPrefix=/v1` set in my application.properties file.  I'm also using the H2 embedded database, when I access /console ( the H2 console ) the Katharsis filter is getting the request and trying to read it.  If I understand your reply correctly it appears this should not be happening because the pathPrefix is /v1 and I'm trying to access /console.  Right?

**Comment by jianglibo** [comment]
Yes, That's what I mean. But you maybe did wrong in somewhere else, Not the Katharsis. check it carefully.

**Comment by adhamhf** [comment]
@jianglibo I've searched and I'm only setting katharsis.pathPrefix in one place.  

@exports a FilterRegistrationBean was a great idea, but setting the order didn't help.  KatharsisFilterV2 eventually gets the request anyway and boom goes the dynamite.

@chb0github I think this is a bug in Katharsis.  Its very easy to reproduce this.  I'm not sure what pathPrefix is being used for, but it does not seem to indicate what path's katharsis should handle.

I have implemented a different filter that reads a katharsis.excludedUrlPaths array property and skips any paths in that property.
